### ISSUE TITLE
make the emzed2_setup.exe portable

### ISSUE THREAD

**Issue by gmat** [uweschmitt/emzed2#22]
Hi,
could you please make the installer .exe portable ?
It just embedded one .bat file...
This would make this program not admin needed.
I've tested it with the portable winpython you've suggest and MS VC++ http://aka.ms/vcpython27 and your .bat file modified.
Thks,

**Comment by uweschmitt** [comment]
What do you mean with "portable"?

**Comment by gmat** [comment]
_portable_ is maybe not the good term in this case. 

Your program could be portable as **mzmine**.  It means you put the folder and launch the .exe every where you want. 

But in your case, with an installer, you really should remove the need for privilege escalation. There's 2 known examples python and R installation on windows. 

For Python (anaconda or python.org), the installer start and first question, "is it just for you or all users ?". First choice == no privilege escalation and suggest an install path in your homedir (first known write dir of this user).

With R, the installer start by a privilege escalation windows as your but if you reply _cancel_ it's continue overall and as python, suggest you a install path in the homedir.

**Comment by uweschmitt** [comment]
I just discovered that the website says you should run the installer with adminstrator privileges. But this is actually not required. I just checked it on a recent Windows machine.

Did you try to install as regular user?

**Comment by gmat** [comment]
oh ! don't know that.
Yes as regular user on windows 10, the installer ask for privilege escalation first no choice if don't enter a administrators users group, it loops. And for _cancel_, it's stop. 
So maybe it's a issue not a request feature. Sorry.

**Comment by uweschmitt** [comment]
I also was not aware that the website is wrong about that!

We are currently working on emzed 3, but due to resource limitations development is slow. After that installation etc will be much easier. Plan is to release next summer.

**Comment by gmat** [comment]
ok thks for the date.
Is there a development goal for the version 3 ? Python 3 I suppose. Will it be "just" a update to make it run on python 3 or will it be based on new concept ? An core spyder independent would be cool.

**Comment by uweschmitt** [comment]
Yes, Python 3, upgrade of PyQt and other libraries, upgrade of spyder. 

Separation into three layers: emzed.lib without any GUI so one can run emzed workflows on HPC, emzed gui for the inspection tools and emzed.spyder for those who want spyder integration.

Beyond that refactoring and cleanup of API, make API PEP8 compliant and also more memory efficient data structures. 

We also plan to integrate some algorithms from mzmine2 as some users reported that peak picking and other functions from mzmine work quite well. This would then be an extension as it requires java to be installed.

So: big plans, but also lots of work ;)
